src/resources/views/crud/inc/datatables_logic.blade.php

```diff
@@ -299,16 +304,16 @@ functionsToRunOnDataTablesDrawEvent: [],
       window.crud.updateUrl(location.href);
 
       // move search bar
```

Comment:
Search bar no longer works for me... it doesn't actually search - no network request, no error, nothing  👀


https://github.com/user-attachments/assets/d8311a1a-fab0-4f89-9ae5-5f1e6df79d01